README.md

```diff
@@ -1 +1 @@
-# village-defense
+# village-defence
```

Comment:
doesnt matter. both are correct xd

Reply:
Maybe, but we should use one version everywhere, which one, it doesn't matter.